permissions/UDisks.permission

```diff
@@ -1,16 +1,19 @@
 # -*- mode: sh -*-
 
-# FIXME: what exactly requires udisks???
-
 # x-sailjail-translation-catalog =
-# x-sailjail-translation-key-description =
-# x-sailjail-description = Use UDisks2
-# x-sailjail-translation-key-long-description =
-# x-sailjail-long-description =
+# x-sailjail-translation-key-description = permission-la-udisks2
+# x-sailjail-description = Manage and access storage devices
+# x-sailjail-translation-key-long-description = permission-la-udisks2_description
+# x-sailjail-long-description = Permission to access the storage devices, allows to call device D-Bus methods.
+
+include /etc/sailjail/permissions/UDisksListen.permission
 
-# BEG systembus-org.freedesktop.UDisks2.resource
-dbus-system.talk org.freedesktop.UDisks2
-dbus-system.broadcast org.freedesktop.UDisks2=org.freedesktop.UDisks2.*@/*
-dbus-system.talk org.freedesktop.UDisks2.*
-dbus-system.broadcast org.freedesktop.UDisks2.*=org.freedesktop.UDisks2.*@/*
-# END systembus-org.freedesktop.UDisks2.resource
+# BEG systembus-org.freedesktop.UDisks2 method call access
+dbus-system.call org.freedesktop.UDisks2=org.freedesktop.DBus.Properties.Set@/*
+dbus-system.call org.freedesktop.UDisks2=org.freedesktop.UDisks2.Block.Format@/org/freedesktop/UDisks2/Block
+dbus-system.call org.freedesktop.UDisks2=org.freedesktop.UDisks2.Block.Rescan@/org/freedesktop/UDisks2/Block
```

Comment:
Are these object paths right?

... I have no idea, but with "qdbus --system org.freedesktop.UDisks2" I get objects like "/org/freedesktop/UDisks2/block_devices/mmcXYZ" that then do have Block interface available.

Reply:
Ah yes, maybe the interfaces could be dropped here as well for the method calls.

Reply:
Because, there are many different paths that make these same calls available.

Reply:
Like e751dcf3128d721bc18314458dcd64e7059e6c4c